Library/Homebrew/unversioned_cask_checker.rb

```diff
@@ -88,8 +98,9 @@ def all_versions
 
         installer.extract_primary_container(to: dir)
 
-        info_plist_paths = apps.concat(qlplugins).flat_map do |artifact|
-          top_level_info_plists(Pathname.glob(dir/"**"/artifact.source.basename/"Contents"/"Info.plist")).sort
+        info_plist_paths = apps.concat(qlplugins, installers).flat_map do |artifact|
+          source = artifact.is_a?(Cask::Artifact::Installer) ? artifact.path : artifact.source.basename
+          top_level_info_plists(Pathname.glob(dir/"**"/source/"Contents"/"Info.plist")).sort
         end
```

Comment:
Is there a reason for this being different than line 151-153 below?

Reply:
No, the logic likely should be the same. I'm not sure how often we are using that method, the only place I could find it was as a part of the `bump-unversioned-casks` dev-cmd.